docs/pages/access-controls/access-request-plugins/index.mdx

```diff
@@ -6,7 +6,34 @@ layout: tocless-doc
 
 Teleport Just-in-Time Access Requests allow users to receive temporary elevated
 privileges by seeking consent from one or more reviewers, depending on your
-configuration. 
+configuration.
+
+With Teleport's Access Request plugins, users can request, approve, and deny
```

Comment:
This isn't the case for most of our plugins like Slack and Teams.  Only JIRA AFAIK can do approve and deny from a user within the outside system.

Reply:
I'll use a more general wording